CoEpi/ui/styles/ButtonStyles.swift

```diff
@@ -0,0 +1,39 @@
+import Foundation
+import UIKit
+
+struct ButtonStyles{
```

Comment:
This LGTM. It's not the fanciest styling system, but it works, until we start using SwiftUI and figure out best practices there.